reports/models.py

```diff
@@ -1 +1,21 @@
-# Create your models here.
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+
+
+@python_2_unicode_compatible
+class ReportTaskStatus(models.Model):
+    """ The status of the report generate tasks
+    """
+
+    start_date = models.CharField(max_length=30, null=False, blank=False)
+    end_date = models.CharField(max_length=30, null=False, blank=False)
+    email_subject = models.CharField(max_length=100, null=False, blank=False)
+    file_size = models.CharField(max_length=30, null=True)
```

Comment:
Is it better to then maybe save the integer and format it where we use it, the CI, the `get_XXX_display` doesn't get called from the view.